Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
```

Comment:
Putting a "%" symbol and a space at the end of the input statement at line 2 would allow for users to know that modulo is an option.

Reply:
Agreed. Simply copying the above format should be sufficient.

Reply:
I agree with you, including the modulo operation is very conducive as it will create multiple options for the user.

Reply:
Agreed, every operation that is available should be made clear to the user, as well, at the moment putting an extra space causes an exception to be thrown, so that should be accounted for

Reply:
I agree as well. Letting the user know that the modulo symbol is an option is a good idea.

Reply:
I agree, having the visual of the "%" symbol would make it very clear for the user that the modulo operator can be used.